[2024-08-21T06:09:07.707+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-21T06:09:07.987+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_22024-08-21.decrypt_the_data scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-08-21T06:09:08.002+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_22024-08-21.decrypt_the_data scheduled__2024-06-02T00:00:00+00:00 [queued]>
[2024-08-21T06:09:08.009+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-21T06:09:10.271+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): decrypt_the_data> on 2024-06-02 00:00:00+00:00
[2024-08-21T06:09:10.420+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_22024-08-21', 'decrypt_the_data', 'scheduled__2024-06-02T00:00:00+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmp9dm7g27p']
[2024-08-21T06:09:10.466+0000] {standard_task_runner.py:91} INFO - Job 276: Subtask decrypt_the_data
[2024-08-21T06:09:10.557+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1528) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T06:09:10.572+0000] {standard_task_runner.py:63} INFO - Started process 1567 to run task
[2024-08-21T06:09:11.002+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_22024-08-21.decrypt_the_data scheduled__2024-06-02T00:00:00+00:00 [running]> on host 7e1333882d71
[2024-08-21T06:09:11.790+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_22024-08-21' AIRFLOW_CTX_TASK_ID='decrypt_the_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-02T00:00:00+00:00'
[2024-08-21T06:09:11.819+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-21T06:09:12.117+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-21T06:09:12.146+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/decryption_utils.py']
[2024-08-21T06:09:12.177+0000] {subprocess.py:86} INFO - Output:
[2024-08-21T06:09:13.545+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-21T06:09:13.619+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-21T06:09:13.857+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_22024-08-21, task_id=decrypt_the_data, run_id=scheduled__2024-06-02T00:00:00+00:00, execution_date=20240602T000000, start_date=20240821T060907, end_date=20240821T060913
[2024-08-21T06:09:14.365+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-21T06:09:14.599+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
